tests/integration-tests/tests/performance_tests/test_starccm/test_starccm/pcluster.config.yaml

```diff
@@ -0,0 +1,42 @@
+Region: {{ region }}
+Image:
+  Os: {{ os }}
+HeadNode:
+  InstanceType: c5n.18xlarge
```

Comment:
HEre you're setting the instance type statically (c5n.18xlarge), but according to the test confg file, this is expected to be customizable.  I suggest you to let the test config file to decide the instance type and inject it in the clister config file.